tt_metal/include/compute_kernel_api/eltwise_binary_sfpu.h

```diff
@@ -61,6 +61,16 @@ ALWI void power_binary_tile(uint32_t idst0, uint32_t idst1) {
 /**
  * Please refer to documentation for any_init.
  */
-ALWI void eltwise_binop_tile_init() { MATH((llk_math_eltwise_binary_sfpu_binop_init<APPROX>())); }
+ALWI void add_binary_tile_init() { MATH((llk_math_eltwise_binary_sfpu_binop_init<APPROX, ADD_BINARY>())); }
+
+ALWI void sub_binary_tile_init() { MATH((llk_math_eltwise_binary_sfpu_binop_init<APPROX, SUB_BINARY>())); }
```

Comment:
So each binary op is potentially a different init.

Reply:
Can we please give the `enum` a name, make it an `enum class`, and use that as the template parameter type instead of `int`?

Reply:
Yes each op needs to be able to have a different init. And ok I will change this to enum class